fastjet/internal/delaunay/point.go

```diff
@@ -46,6 +46,10 @@ func (p *Point) Coordinates() (x, y float64) {
 	return p.x, p.y
 }
 
+func (p *Point) ID() int {
```

Comment:
do we need to export this?
in that case, please document that exported method.

Reply:
done

Reply:
And yes we need to export it because we need a way of mapping the points back to the jets